content/en/docs/tasks/inject-data-application/define-command-argument-container.md

```diff
@@ -36,7 +36,7 @@ If you define args, but do not define a command, the default command is used
 with your new arguments.
 
 {{< note >}}
-The `command` field corresponds to `entrypoint` in some container runtimes.
+The `command` field corresponds to `ENTRYPOINT`, `args` field corresponds to `CMD` in some container runtimes.
```

Comment:
I would suggest a small grammatical change

```suggestion
The `command` field corresponds to `ENTRYPOINT`, and the `args` field corresponds to `CMD` when `ENTRYPOINT` is used.
```

Reply:
I remain [and the] to fix grammatical error. and use [in some container runtimes] before.